src/classes/ADDR_Contact_TEST.cls

```diff
@@ -0,0 +1,1112 @@
+        for (Integer i = 0; i < contacts.size(); i++) {
+            addrs[i].Parent_Contact__c = contacts[i].Id;
+            addrs[i].Default_Address__c = true;
+        }
+        ADDR_Addresses_TDTM.hasRunAddrTrigger = false;
+        insert addrs;
+        ADDR_Addresses_TDTM.hasRunAddrTrigger = false;
+        
+        // verify results
+        contacts = [select Id, Name, MailingStreet, MailingCity, MailingState, 
+        MailingPostalCode, MailingCountry, MailingLatitude, MailingLongitude, is_Address_Override__c, Current_Address__c 
+        from Contact order by Contact.Name];
+        system.assertEquals(3, contacts.size());
+        
+        addrs = [select Id, Default_Address__c, MailingStreet__c, MailingCity__c, Parent_Contact__c, 
+        Latest_Start_Date__c, Latest_End_Date__c from Address__c order by Parent_Contact__r.Name];
+        system.assertEquals(3, addrs.size());
+        
+        for (Integer i = 0; i < contacts.size(); i++) {
+            Contact contact = contacts[i];
+            Address__c addr = addrs[i];
+            system.assertEquals(contact.MailingStreet, addr.MailingStreet__c);
+            system.assertEquals(contact.MailingCity, addr.MailingCity__c);
+            system.assertNotEquals(null, contact.MailingStreet);
+            system.assertNotEquals(null, contact.MailingCity);
+            system.assertEquals(true, addr.Default_Address__c);
+            system.assertEquals(addr.Id, contact.Current_Address__c);
+        }
+    }
+
+    /*********************************************************************************************************
+    @description Create Contacts when the setting for Address Mgmt is off. 
+    verify:
+        no Address objects created
+        Contact addresses set
+    **********************************************************************************************************/            
+    @isTest
+    public static void testDisabledAddr() {        
+        UTIL_CustomSettingsFacade.getSettingsForTests(new Hierarchy_Settings__c(Contacts_Addresses_Enabled__c = false));
+
+        List<Contact> contacts = UTIL_UnitTestData_TEST.CreateMultipleTestContacts(2);
+        for (Contact contact : contacts) {
+            contact.MailingStreet = 'new street';
+            contact.MailingCity = 'new city';
+        }
+
+        // verify Contact address
+        map<Id, Contact> mapAccIdAcc = new map<Id, Contact>([select Id, Name, MailingStreet, MailingCity, MailingState, 
+        MailingPostalCode, MailingCountry, MailingLatitude, MailingLongitude, is_Address_Override__c, Current_Address__c from Contact]);
+        
+        for (Contact contact : mapAccIdAcc.values()) {
+            system.assertEquals(false, contact.is_Address_Override__c);
+            system.assertEquals(null, contact.Current_Address__c);
+        }
+        
+        // verify no address objects created
+        List<Address__c> ListAddr = [select Id from Address__c];
```

Comment:
Variables should use the `lowerCamelCase` naming convention. Please rename accordingly.